test/unit/health.test.ts

```diff
@@ -12,15 +12,15 @@ describe("health", () => {
   })
 
   it("/healthz", async () => {
-    ;[, , codeServer] = await integration.setup(["--auth=none"], "")
+    codeServer = await integration.setup(["--auth=none"], "")
```

Comment:
this looks way cleaner/easier to read 👏

Reply:
:+1: yea, the code before was :raised_eyebrow: kinda inscrutable, but then again, I'm not a JavaScript/TypeScript person